ansible/playbooks/adhoc/grow_root_vg/filter_plugins/oo_filters.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
```

Comment:
The filename `oo_filters.py` may cause a problem. Please see @mwoodson email regarding Ansible 2.4 and duplicate filter names. I believe it just needs to be unique.

Reply:
If this is a copy of a filter, perhaps we can make it globally available to all playbooks instead of adding a copy of it?

Reply:
I moved this to ../../../filter_plugins and renamed it, but ansible-playbook didn't find the filters until I added a symlink back.